src/main/resources/mixins.common.optimization.json

```diff
@@ -11,6 +11,12 @@
         "block.MixinBlockRedstoneWire",
         "enchantment.MixinEnchantmentHelper_No_Source_Leak",
         "entity.MixinEntityTameable_Cached_Owner",
+        "mapoptimization.MixinEntityItemFrame",
```

Comment:
I personally prefer the suffixed names so open class lookups aren't so confusing or annoying to have to read the fine print to know which one is which.